package.json

```diff
@@ -36,6 +37,7 @@
     "google-oauth-jwt": "0.1.7",
     "googleapis": "0.8.0",
     "lodash": "2.4.1",
+    "meetup-api": "^1.4.9",
```

Comment:
I think that's a really bad approach, not to mention that it's ineffective. Almost all packages that are in Node ecosystem use semver approach. Even if we pin a dependency to be a fixed version, it's dependencies might not be fixed. Moreover, by using fixed dependency we are saying no to security and performance patches for that package.